src/game/server/entities/looper-wall.cpp

```diff
@@ -143,13 +144,23 @@ void CLooperWall::Snap(int SnappingClient)
 			CNetObj_Laser *pObj = static_cast<CNetObj_Laser *>(Server()->SnapNewItem(NETOBJTYPE_LASER, m_IDs[i], sizeof(CNetObj_Laser))); //removed m_ID
 			if(!pObj)
 				return;
+			
+			if (i == 0) 
+			{
+				pObj->m_X = (int)m_Pos.x+dirVec.y*(THICKNESS*0.5f); 
+				pObj->m_Y = (int)m_Pos.y-dirVec.x*(THICKNESS*0.5f);
+				pObj->m_FromX = (int)m_Pos2.x+dirVec.y*(THICKNESS*0.5f); 
+				pObj->m_FromY = (int)m_Pos2.y-dirVec.x*(THICKNESS*0.5f);
+			} 
+			else // i == 1
+			{
+				pObj->m_X = (int)m_Pos.x-dirVec.y*(THICKNESS*0.5f); 
+				pObj->m_Y = (int)m_Pos.y+dirVec.x*(THICKNESS*0.5f);
+				pObj->m_FromX = (int)m_Pos2.x-dirVec.y*(THICKNESS*0.5f); 
+				pObj->m_FromY = (int)m_Pos2.y+dirVec.x*(THICKNESS*0.5f);
+			}
```

Comment:
I realized i can avoid the if else by multiplying by i*-1, saving several lines of code.
But at the same time the code would become a tiny bit less efficient and a bit harder to understand.
I think less code is better in this case, but I dont know how to change this now.
I might add a the pistol gun particle effect in the future and also change this code while doing it.